src/components/__tests__/image_upload.test.tsx

```diff
@@ -1,26 +1,42 @@
 // Copyright (c) 2018 Ultimaker B.V.
-import 'jest';
-import 'jsdom-global/register';
 import * as React from 'react';
 import { shallow } from 'enzyme';
 
 // component
 import ImageUpload from '../image_upload';
 
 describe('The ImageFile component', () => {
-  let props;
-  let wrapper;
-
-  beforeEach(() => {
-    props = {
-      onFileSelection: jest.fn(),
-    };
-    wrapper = shallow(<ImageUpload {...props} />);
-  });
-
-  it('should render', () => {
-    expect(wrapper).toMatchSnapshot();
-    expect(props.onFileSelection).not.toHaveBeenCalled();
-  });
+    let props;
+    let wrapper;
+
+    beforeEach(() => {
+        props = {
```

Comment:
there is an issue with react-dropzone that is generating an error:
https://github.com/react-dropzone/react-dropzone/issues/554

Reply:
updated